src/main/java/hudson/remoting/Launcher.java

```diff
@@ -283,12 +356,20 @@ public void setNoCertificateCheck(boolean ignored) throws NoSuchAlgorithmExcepti
     @Option(name="-version",usage="Shows the version of the remoting jar and then exits")
     public boolean showVersion = false;
 
+    /**
+     * The original calling convention takes two positional arguments: secret key and agent name.
+     * @deprecated use {@link #secret} and {@link #name}
+     */
+    @Argument
+    @Deprecated
+    public List<String> args = new ArrayList<>();
```

Comment:
The other entrypoint took two positional arguments, one for the secret and one for the agent name. Confusingly enough, the secret can also be provided by the server if using `-jnlpUrl` and `-credentials`. In any case, we now have named arguments for these for consistency with everything else, so deprecating the positional arguments.